.travis.yml

```diff
@@ -4,7 +4,11 @@ python:
   - "2.7"
 
 install:
+    - pip install flake8
     - pip install -r requirements.txt
 
+before_script:
+    - flake8 .
```

Comment:
I think it is better to add flake8 to the script section - this is a setup step, but an actual test, and we want it to behave just like the pytest invocation.

Also, we like to use these options:

    --show-source         Show the source generate each error or warning.
    --statistics          Count errors and warnings.